sympy/core/expr.py

```diff
@@ -2147,12 +2147,21 @@ def as_numer_denom(self):
 
         See Also
         ========
-        normal: return a/b instead of a, b
-        """
 
+        normal: return ``a/b`` instead of ``(a, b)``
+
+        """
         return self, S.One
 
     def normal(self):
+        """ expression -> a/b
+
+        See Also
+        ========
+
+        as_numer_denom: return ``(a, b)`` instead of ``a/b``
+
+        """
         from .mul import _unevaluated_Mul
         n, d = self.as_numer_denom()
         if d is S.One:
```

Comment:
@oscarbenjamin 
I think you may want to take a look at this change. Previously, `See Also` in `Expr.as_numer_denom()` linked to `Q.normal`, not `Expr.normal` because it had no docstring. I added a succinct docstring to fix this.